src/Core/Services/IReferenceEventService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Threading.Tasks;
+using Bit.Core.Models;
+
+namespace Bit.Core.Services
+{
+    public interface IReferenceEventService
+    {
+        Task RaiseEventAsync(IReferenceable reference, string eventType,
```

Comment:
Should events be defined as enums similar to our eventservice?

Reply:
That would make sense, I'll make that change.